nextflow.config

```diff
@@ -254,7 +254,7 @@ manifest {
     description     = 'B and T cell repertoire analysis pipeline with the Immcantation framework.'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=21.10.3'
-    version = '2.3.0dev'
+    version = '2.3.0'
```

Comment:
like always, nf-core bump-version doesnt align here nicely. I usually correct that, but sometimes forget it ;)

Reply:
ah good point

Reply:
I think I can't be bothered that all tests need to run again though...

Reply:
Wow that one test profile runs >1h 20min, indeed, that alignment isnt worth that :D